plugins/connection/incus.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+# Based on lxd.py (c) 2016, Matt Clay <matt@mystile.com>
+# (c) 2023, Stephane Graber <stgraber@stgraber.org>
+# Copyright (c) 2023 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    author: Stéphane Graber <stgraber@stgraber.org>
+    connection: incus
+    short_description: Run tasks in Incus instances via the Incus CLI
+    description:
+        - Run commands or put/fetch files to an existing Incus instance using Incus CLI
+    version_added: "2.0"
+    options:
+      remote_addr:
```

Comment:
This can be confusing as `project + instance_name` is unique but the instance_name itself is not not unique across projects but for Ansible `inventory_hostname` is the unique identifier.

Does it make sense to have  `instance_name@project` as the `inventory_hostname` and then split on that? 
or are most instances going to be `inventory_hostname`(instance name) at `default` (project)?

Reply:
this is probably something to be addressed with the inventory plugin also.

Reply:
I suspect most will be inventory_hostname at default project. It's only when you're dealing with larger scale deployments that the projects come at play.

Reply:
k, leave as is for now, we can address on inventory when `project !=default` and/or `remote != local`. We can come up with conventions there for `inventory_hostname` and just populate the `ansible_incus_` variables.